project/settings.py

```
@@ -27,24 +27,71 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/5.2/howto/deployment/checklist/
 
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-vszkevrb2_f(+1hu1_ioq%mwg!h#w!if#ucg83x6wwq_s(&e5j"
+SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY')
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)